### ISSUE TITLE
Implement the placement of danger points

### ISSUE THREAD

**Issue by jetwell** [jetwell/adfc-radschulwegplan-frontend#2]
## Acceptance Criteria
* A button exists to initiate the placement of a danger point
* The user is given feedback, that the danger point placement feature is active
* The danger point can be placed with a click / tap interaction
* The danger point can be moved after placement with a drag and drop interaction
* A dialogue appears on placement to enter the title, type and description of the danger point

**Comment by tabacha** [comment]
Here an example, how to drag a marker https://gis.stackexchange.com/questions/80306/leaflet-draggable-marker

**Comment by tabacha** [comment]
I like smaler issues. Perhabs it is better to split this issue into smaler parts?

**Comment by jetwell** [comment]
Yes, we should definitely split this one up. I would suggest moving the dialogue and drag and drop features to separate tickets.
